extensions-contrib/redis-cache/src/main/java/org/apache/druid/client/cache/RedisClusterCache.java

```diff
@@ -110,10 +109,6 @@ protected Pair<Integer, Map<NamedKey, byte[]>> mgetFromRedis(Iterable<NamedKey>
   @Override
   protected void cleanup()
   {
-    try {
-      cluster.close();
-    }
-    catch (IOException ignored) {
-    }
+    cluster.close();
```

Comment:
why this change?

Reply:
Thank you for the review @abhishekagarwal87! In Jedis 5.0.2, that method is inherited from the `UnifiedJedis` class and [it doesn't raise IOException since `IOUtils.closeQuietly` is internally used](https://github.com/redis/jedis/blob/v5.0.2/src/main/java/redis/clients/jedis/UnifiedJedis.java#L233-L236).